Lets Do This/Controllers/Campaign/LDTCampaignDetailViewController.m

```diff
@@ -166,24 +168,27 @@ - (void)configureCampaignCell:(LDTCampaignDetailCampaignCell *)cell {
     cell.solutionCopyLabelText = self.campaign.solutionCopy;
     cell.solutionSupportCopyLabelText = self.campaign.solutionSupportCopy;
     cell.coverImageURL = self.campaign.coverImageURL;
+    cell.campaignDetailsHeadingLabelText = @"Do this".uppercaseString;
+    cell.staticInstructionLabelText = @"When you’re done, submit a pic of yourself in action. #picsoritdidnthappen";
+
     if ([[self user] hasCompletedCampaign:self.campaign]) {
-        cell.displaySubmitReportbackButton = NO;
+        cell.displayActionButton = NO;
+        cell.displayCampaignDetailsView = YES;
```

Comment:
If I remove this line (which you wouldn't think you'd need since the `campaignDetailView` should already be displayed here) -- the constraints don't break but the `diagonalShapeLayer` is missing and the constraints / layout is incorrect.